example/crs/mellanox.com_v1alpha1_nicclusterpolicy_cr-full.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2020 NVIDIA
```

Comment:
Please, update copyright header as at [1] and fix the year

[1] https://github.com/Mellanox/network-operator/blob/6800708dee1bdd073b31a24660c74fe509b55735/pkg/migrate/migrate.go#L1

Reply:
Done